product/en-US/development_guide/important_log_messages.xml

```diff
@@ -56,4 +56,26 @@
             </para>
         </important>
     </section>
+    <section>
+        <title>Multi cause log message</title>
+        <para>The following table shows some log messages that you may see with an explanation of alternate reasons</para>
+        <informaltable>
+            <tgroup cols="2">
+                <tbody>
+                    <row>
+                        <entry>
+                            <para>INFO  [com.arjuna.ats.arjuna] ObjectStore record was deleted during restoration, users should not deleted records manually: ***/ShadowNoFileLockStore/defaultStore/StateManager/BasicAction/TwoPhaseCoordinator/AtomicAction/SubordinateAtomicAction/JCA/***: java.io.FileNotFoundException: ***/ShadowNoFileLockStore/defaultStore/StateManager/BasicAction/TwoPhaseCoordinator/AtomicAction/SubordinateAtomicAction/JCA/*** (No such file or directory)</para>
```

Comment:
It's not fatal in so far as the user intended to delete the record so they intended the behavior to happen as they had consulted the manual so no the output to look for in that scenario